src/bindings/python/src/openvino/_ov_api.py

```diff
@@ -43,7 +43,8 @@ def __getattr__(cls, name: str) -> Any:
 class Model(object, metaclass=ModelMeta):
     def __init__(self, *args: Any, **kwargs: Any) -> None:
         if not args and not kwargs:
-            self.__model = ModelBase()
+            raise RuntimeError("Cannot initialize Model with no arguments.")
```

Comment:
https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/python/tests/test_runtime/test_model.py#L914

You also need to update this test